doc/man-sections/client-options.rst

```diff
@@ -87,6 +87,20 @@ configuration.
   The server configuration must specify an ``--auth-user-pass-verify``
   script to verify the username/password provided by the client.
 
+--external-auth
+  For security reasons, OpenVPN requires client-side credentials such as
+  client certificates or a username/password combination. The OpenVPN server
+  has the capability to delegate authentication to external systems using the
+  WEBAUTH protocol. In such cases, client credentials may be omitted.
+
+  ***Security Considerations***
+
+  When the ``--external-auth`` option is enabled in OpenVPN, it bypasses the
+  standard authentication checks. This configuration can potentially create a
```

Comment:
"the standard authentication checks" could be misleading. What about "the check that some form of user authentication method is specified." 

That said, no matter how carefully this is worded, its going to be ambiguous as the  whole rationale for the client-side check is no unnatural and unnecessary. Best route would have been to completely omit the check -- hope repeating makes a wish come true ;)